app/views/decidim/challenges/challenges/show.html.erb

```diff
@@ -1,6 +1,11 @@
 <%= stylesheet_link_tag "decidim/sdgs/ods" %>
 <%= stylesheet_link_tag "decidim/challenges/challenges" %>
 
+<% associated_problems_list = @challenge.problems %>
+<% proposed_solutions_list = [] %>
+<% associated_problems_list.each do |problem| %>
+  <% proposed_solutions_list.concat problem.solutions %>
+<% end %>
```

Comment:
why do we need this?
`proposed_solutions_list` is only used once so we can directly use `@challenge.problems` no?

Reply:
I could be possible that instead of `proposed_solutions_list` you were referring to `associated_problems_list` @tramuntanal ?